pkgs/applications/misc/pagefind/default.nix

```diff
@@ -88,13 +106,20 @@ rustPlatform.buildRustPackage rec {
     rustc.llvmPackages.lld
     wasm-bindgen-92
     wasm-pack
+    httplz
   ];
 
   # build wasm and js assets
   # based on "test-and-build" in https://github.com/CloudCannon/pagefind/blob/main/.github/workflows/release.yml
   preBuild = ''
     export HOME=$(mktemp -d)
 
+    # serve lindera-unidic on localhost
+    mkdir .lindera-http-plz
+    ln -s ${lindera-unidic-src} .lindera-http-plz/unidic-mecab-2.1.2.tar.gz
+    httplz --port 34567 -- .lindera-http-plz/ &
```

Comment:
Does it make sense to hardcode 34567? can we use a port assigned by the os? i.e. 0
Then serve the file using httplz in postPatch before substituing the url?

sent a pr, https://github.com/pbsds/nixpkgs/pull/1

Reply:
It is safe within the linux sandbox, but not so much on darwin. i'll look at the PR